Godeps/Godeps.json

```diff
@@ -406,6 +426,11 @@
 			"Rev": "a1b8ba5163b7f041b22761461eabd02b70d1f824"
 		},
 		{
+			"ImportPath": "github.com/gogo/protobuf/proto",
+			"Comment": "v0.1-125-g82d16f7",
+			"Rev": "82d16f734d6d871204a3feb1a73cb220cc92574c"
```

Comment:
I don't know if https://github.com/gogo/protobuf/issues/182 would be done in time.

Does it actually fail with the old tag `v0.2`?
https://github.com/coreos/rkt/pull/2673#discussion_r64388932